draft-ietf-opsawg-pcap.md

```diff
@@ -292,24 +310,20 @@ specification.
 
 Please note: To avoid confusion (such as the current usage of .cap for a
 plethora of different capture file formats) file name extensions other
-than .pcap should be avoided.
+than `.pcap` should be avoided.
 
-There is new work to create the PCAP Next Generation capture File Format
-(see {{I-D.ietf-opsawg-pcapng}}).  The new file format is not
-compatible with this specification, but many programs read both
-transparently.  Files of that type will start with a Section
-Header Block, the first four octets of which are 0x0A 0x0D 0x0D 0x0A,
-which does not match any of the Magic Number values in a pcap File
-Header, allowing code that reads both file formats to determine the
-format of a file.
+A new format that supersedes the present PCAP format is specified in {{I-D.ietf-opsawg-pcapng}} (referred to as "new PCAP format").
+The new file format is not compatible with this specification, but many programs read  both transparently.
+Version numbers in the two file formats are not related.
+A distinct Magic Number exists in the new PCAP format so that readers can transparently determine what format is used.
+It is not uncommon for the same extension, `.pcap` to be used for both.
 
 #  Security Considerations
 
-A pcap file reader MUST do invalid header and packet checks.
-It can receive as input not only valid headers or packets, but any arbitrary
+A pcap file reader MUST do validate the file header and file packet header, and also the contained headers for the packet capture.
+A reader can receive as input not only valid headers or packets, but any arbitrary
 random sequence of octets:
-Headers or packets originally malformed by the sender or by a fuzz tester,
-corrupted in transit or for some other reason.
+Headers or packets originally may be intentionally malformed by a sender. Also, capture files from outside sources may contain intentionally malformed contents.
```

Comment:
```suggestion
Headers or packets may be intentionally malformed by a sender., and capture files from outside sources may contain intentionally malformed contents, for malicious reasons.
```

Reply:
Remove the "." in the new version after "sender" and before ",".

Reply:
> Remove the "." in the new version after "sender" and before ",".

You ought to be able to hit edit on my suggestion, but I'll do that.